pkg/serverless/daemon/daemon_test.go

```diff
@@ -178,3 +178,13 @@ func TestSetTraceTagOk(t *testing.T) {
 	}
 	assert.True(t, d.setTraceTags(tagsMap))
 }
+
+func TestOutOfOrderInvocations(t *testing.T) {
```

Comment:
You can use [debug.SetPanicOnFault](https://pkg.go.dev/runtime/debug#SetPanicOnFault) to recover from a SIGSEGV properly, in case you wanted to run that test without the above codechange to prove it works. E.g. pkg/serverless/serverless_test.go:63